src/test/java/org/kainos/ea/service/ValidatorServiceTest.java

```diff
@@ -0,0 +1,106 @@
+package org.kainos.ea.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.cli.Login;
+import org.kainos.ea.cli.Register;
+import org.kainos.ea.core.Validator;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.kainos.ea.cli.Role.Admin;
+import static org.kainos.ea.cli.Role.Employee;
+
+@ExtendWith(MockitoExtension.class)
+public class ValidatorServiceTest {
+    private Validator validator;
+
+    @BeforeEach
+    public void setUp() {
+        validator = Mockito.mock(Validator.class);
+    }
+
+    @Test
+    public void testIsValidLogin_EmailTooLong() {
+        Login login = new Login("ssssssssssssssssssandjbfsdhjbfdhsjbfhjsdbfhjdsbfjhbsdhfjbsdhjfbsdhjfbdsjhfbsdjhbfdshjfbjhsdbf@example.com", "password123");
+        Mockito.when(validator.isValidLogin(login)).thenReturn("Email greater than 64 characters");
```

Comment:
you're mocking out the validator class when you're supposed to be testing your validator class code, remove the mock and call the validator directly